easybuild/easyconfigs/n/nvtop/nvtop-3.0.1-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'nvtop'
+version = '3.0.1'
+
+homepage = 'https://github.com/Syllo/nvtop'
+description = 'htop-like GPU usage monitor'
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = ['https://github.com/Syllo/nvtop/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['f61ecbe3a30a216706c16fc177d3b840e7d7deccf87b048d59f95f230d591a53']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('CMake', '3.24.3'),
+]
+
```

Comment:
```suggestion

osdependencies = [('libsystemd-dev', 'systemd-devel')]

```

Reply:
Shouldn't `libsystemd-dev` and `systemd-devel` be enough considering we can build `libdrm` ourselves?